tests/openvino/tools/calibrate.py

```diff
@@ -220,7 +220,7 @@ def transform_fn(data_item):
         _, batch_annotation, batch_input, _ = data_item
         filled_inputs, _, _ = model_evaluator._get_batch_input(
             batch_input, batch_annotation)
-        return filled_inputs
+        return filled_inputs[0]
```

Comment:
Can we be sure that `model_evaluator._get_batch_input` always returns a list as the first output? Can't we take the first batch this way, for example?

Reply:
That's a good point. I will check it.

Reply:
> Can we be sure that model_evaluator._get_batch_input always returns a list as the first output?

Yes, we can.

> Can't we take the first batch this way, for example?

The `filled_inputs` is a `List[Dict[str, np.ndarray]]`. Probably `len(filled_inputs)` always equals to 1.